### PR TITLE
Hello I'm Max

### PR DESCRIPTION
**Pull request by phpslonuk**
I'm with you, finished the course Version Control with Git

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ P.S. Where all postings before March 2018 have gone to?
 It was an explosive start to front-end learning! At first it was hard for me because of English)) but with each lesson it became easier and easier to learn. I really liked the Git tool, and I will definitely use it in my studies, work and my projects. Many thanks to Kottans for the great course. I'm moving on to the next Linux, Command Line, HTTP tools!)
 
 ---
+# Hi, everyone :)
+I’m [Max](https://github.com/phpslonuk/kottans-frontend)
+![All of us](https://i.pinimg.com/originals/1c/26/fb/1c26fbf728599191de9defc63e9ac2a9.gif)
+_________________________________________________
```
Comment (babayK0) [inline README.md:52]: Add your text in the end of README.md

**Review by babayK0 — CHANGES_REQUESTED** [review]
Add your text in the end of README.md

**Comment by phpslonuk** [comment]
Changed README.md

File: README.md [hunk 2]
```diff
@@ -1741,6 +1741,13 @@ I see there is already one photo of my cat, but it can't be too much of him, rig
 I'm [Evgeniy](https://github.com/Evgeniy241984)! 
 Glad to join!
 
+<<<<<<< HEAD
+_________________________________________________
+
```
Comment (babayK0) [inline README.md:1746]: Seems like you forgot to delete vscode(or other editors') "conflict pointers"

File: README.md [hunk 3]
```diff
@@ -1741,6 +1741,13 @@ I see there is already one photo of my cat, but it can't be too much of him, rig
...
+# Hi, everyone :)
+I’m [Max](https://github.com/phpslonuk/kottans-frontend)
+![All of us](https://i.pinimg.com/originals/1c/26/fb/1c26fbf728599191de9defc63e9ac2a9.gif)
+=======
```
Comment (babayK0) [inline README.md:1750]: Here same

File: README.md [hunk 4]
```diff
@@ -1752,3 +1759,4 @@ I'm [Serge](https://github.com/sergesd). It's really nice to get new and importa
 
 ## Hi! I'm Sergey, and here is Shakespeare, he's very british :)
 ![cat-image](assets/images/shakespeare.jpg)
+>>>>>>> master
```
Comment (babayK0) [inline README.md:1762]: Same

**Review by babayK0 — CHANGES_REQUESTED** [review]

**Comment by phpslonuk** [comment]
Deleted 'conflict pointers'

**Review by babayK0 — APPROVED** [review]

**Review by babayK0 — APPROVED** [review]

**Pull request merged**